### ISSUE TITLE
Accessibility testing violation by carbon components

### ISSUE THREAD

**Issue by pkegbeto** [carbon-design-system/carbon#382]
## Detailed description

I am running accessibility test using @ibma/aat against a nodejs/react app. I am getting some violations about snippets rendered from Carbon components

> Is this issue related to a specific component? 

It is pervasive

> What did you expect to happen? What happened instead? What would you like to see changed?

Expect to happen: Pass the accessibility testing
Happened instead: Failed the accessibility testing

> What browser are you working in? 

Chrome webdriver

>What version of the Carbon Design System are you using? 

7.19.1

## Steps to reproduce the issue

1. Run the Automated Accessibility Testing 
2. Compare result with accessibility guidelines

**Comment by tw15egan** [comment]
Could you upload an image of the errors you're receiving?

**Comment by pkegbeto** [comment]
<img width="713" alt="screen shot 2017-10-24 at 3 04 13 pm" src="https://user-images.githubusercontent.com/29408678/31962891-d96f1d9e-b8cc-11e7-9d1f-fe0f677ba2d4.png">
<img width="701" alt="screen shot 2017-10-24 at 3 04 41 pm" src="https://user-images.githubusercontent.com/29408678/31962892-d97b6360-b8cc-11e7-93eb-96957765da1b.png">
<img width="715" alt="screen shot 2017-10-24 at 3 05 12 pm" src="https://user-images.githubusercontent.com/29408678/31962893-d9882aaa-b8cc-11e7-87a1-95acd4b795e4.png">

**Comment by pkegbeto** [comment]
Done!

Regards.
____________________________________
Koku Patrick Egbetoke
Cloud Services Developer - GTS
2455 South Road
Poughkeepsie, NY 12601
Tel: 917-421-8403  (T/L 2-362-8403)
Email: pkegbeto@us.ibm.com

**Comment by marijohannessen** [comment]
Hi @pkegbeto , can you please add more descriptive messages or explain what the error messages mean? It's impossible for us to decode your screenshot without any explanations. It looks like this is all related to the interior left nav? What specifically is wrong with it? Thanks!

**Comment by carmacleod** [comment]
@pkegbeto 
1) Please paste a link to the page (or pages) that caused this list of violations.
2) Also, please paste the list of violations as text, and not an image, so that the help links can be followed.

**First**, however: It would be useful (and instructive) if you could look at each violation on your own.
For example, based on the very cryptic output image, I see that:
- you have an h1 with no text (it should have text content, or an image with alt text, or if neither of those is possible, it should have an aria-label).
- your `<html>` is missing `<html lang="en">` (or whatever other language the page will be rendered in).
- you have several anchor links that are missing text (again, they need either text content, or an image with alt text, or if neither of those is possible, they should have an aria-label).
- you have anchor links that fail the color contrast ratio requirement of 4.5:1 for foreground text to background color. (this one may or may not be due to Carbon color choices - I don't know - you need to decide).
- you have menu items and a menu without labels (again, they need either text content, or an image with alt text, or if neither of those is possible, they should have an aria-label). This may or may not be a Carbon problem, but before reporting, you should look to see whether you just need to add a visible or invisible label.
- you have an "orphaned" h1, meaning that it isn't inside a landmark region (main, header, footer, nav, aside, form, section; note that form and section must have a label to be a landmark, nav and aside need labels if there's more than one, and only the top level main, header, footer are considered landmarks).

Please note also that violations related to SVG elements and their components (such as the `<title>` element) can be ignored (for now) until support for SVG is added to the tool. Here is the [DAP/RPT Epic to support SVG2](https://github.ibm.com/IBMa/Tools-Rules-HTML/issues/6).

If you have not already done so, please run the Chrome DevTools Plugin called Dynamic Assessment Plugin (DAP) on your pages. This is the same set of rules as the AAT, but in an interactive format. Here are the [DAP installation instructions and usage](http://ibm.biz/A11yToolsLandingPage).

**Comment by pkegbeto** [comment]
Hi,

The link to the page is: 
https://cmsportal.w3ibm.mybluemix.net/

I compiled some of the violations here:


[aat-errors.txt](https://github.com/carbon-design-system/carbon-components/files/1488833/aat-errors.txt)
